team_admin/discord.md

```diff
@@ -0,0 +1,28 @@
+---
+layout: page
+title: Discord Server
+---
+
+# Discord Server
+
+To assist with communications, especially in a virtual competition, there's a [Discord](https://discord.com/) communication both for communication with blue-shirts and among your team.
+
+If you do not wish to, or are unable to join Discord, you can still get support from <{{ site.emails.teams }}> as normal.
+
+## Joining
+
+Invite links will be sent to team leaders. Only competitors should be given access.
+
+Once invited, you'll only have access to a few channels, namely `#welcome-and-rules` and `#welcome-<username>` (with your username in). Before you can join, you must read the rules shown in the `#welcome-and-rules` channel, and enter the password for your team in your personal `#welcome-<username>` channel.
+
+## Channels
+
+The server has a number of channels for specific conversation. Please try to keep channels on-topic, and move channels if the conversation tangents. Each channel has a topic at the top of the screen showing what it's about.
+
+### Team channels
+
+Along with the common channels, there are some channels visible only to members of your team.
+
+`team-ABC` (where `ABC` is your TLA) is a channel only your team (and blue-shirts) can see. Here you can converse among your team, and get help from blue-shirts without giving away too much of your strategy. These channels should be considered your own, and can be used to discuss anything competition related for your team.
+
+If your school / institution has multiple teams (`ABC2` etc), there will be an additional channel for each of those (`teams-ABC2`). Remember to try and keep conversations specific to the team - Just because the team comes from the same school / institution, doesn't make them any less your rival.
```

Comment:
Grammar:
```suggestion
If your school / institution has multiple teams (`ABC2` etc), there will be an additional channel for each of those (`teams-ABC2`). Remember to try and keep conversations specific to the team - just because the team comes from the same school / institution, doesn't make them any less your rival.
```
Or break this into two sentences.